app/schemas/blog.py

```diff
@@ -8,18 +9,19 @@ class BlogBase(BaseModel):
 
 
 class BlogCreate(BlogBase):
-    owner_id: int
+    pass
 
 
 class BlogUpdate(BlogBase):
-    is_published: bool
+    title: Optional[str] = None
+    content: Optional[str] = None
+    tags: Optional[str] = None
+    is_published: Optional[bool] = None
```

Comment:
is their any case where the is_published have to set the None, in that case is it will be treated as published or not?